# common params
run_factor: 2.0

# buttons
deadman_button: 5
run_button: 7

# axes
axis_vx: 1
axis_vy: 0
axis_vth: 2
up_down: 5 #tilt--up/down; tray--front/back, here we just name up_down
left_right: 4 #pan--left/right

